notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb

```diff
@@ -0,0 +1,1059 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    import tensorflow as tf
</pre><p>Remove TF dependence and replace with gsutil. </p><p> </p><p>If it's cleaner, use google-cloud-storage API.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>